src/test/java/net/bramp/ffmpeg/progress/StreamProgressParserTest.java

```diff
@@ -13,17 +15,20 @@
 
 public class StreamProgressParserTest {
 
-  RecordingProgressListener listener = new RecordingProgressListener();
+	RecordingProgressListener listener = new RecordingProgressListener();
 
-  @Test
-  public void testNormal() throws IOException {
-    listener.reset();
+	@Before
+	public void setup() throws IOException {
+		listener.reset();
 
-    StreamProgressParser parser = new StreamProgressParser(listener);
+		StreamProgressParser parser = new StreamProgressParser(listener);
 
-    InputStream inputStream = combineResource(Progresses.allFiles);
-    parser.processStream(inputStream);
+		InputStream inputStream = combineResource(Progresses.allFiles);
+		parser.processStream(inputStream);
```

Comment:
These two lines directly relate to the thing being test. So they should be in the test, not the setup.

If were to write another test, what from the setup would we need to reuse?

Reply:
_These two lines directly relate to the thing being test. So they should be in the test, not the setup._
Once, in the original code, line 24 instantiates an object which is necessary to perform the test, and an exception might be raised, the line should be placed in the test setup. Line 25 is indeed the unit under test and should remain in the test method. My commit fails to accomplish that. I am sorry.

_If were to write another test, what from the setup would we need to reuse?_
If the new test can not use the already created setup steps, then a test suite should be designed, with every test having its setup steps and the test suite having the shared setup steps, if any.